src/client/client4.js

```diff
@@ -1071,8 +1069,6 @@ export default class Client4 {
     };
 
     removeFromChannel = async (userId, channelId) => {
-        this.trackEvent('api', 'api_channels_remove_member', {channel_id: channelId});
```

Comment:
I don't think we should remove these, as this is tracking API usage and the new ones are tracking specific actions